README.md

```diff
@@ -1,9 +1,9 @@
 <h1 align="center">mapr</h1>
 <p align="center">
-  <img width="200px" alt="Logo" src="https://">
+  <img width="200px" src="https://static-assets.mapbox.com/www/logos/mapbox-logo-black.png" alt="Mapbox Logo">
```

Comment:
The `alt` attribute was moved to the end to match the preview image element following it.

Reply:
I'll create a placeholder logo for MapLibre+Rust today. I would like to avoid to link to any mapbox resources.